metacat-main/src/main/java/com/netflix/metacat/main/api/v1/MetacatController.java

```diff
@@ -956,6 +957,7 @@ public void updateDatabase(
             () -> {
                 final DatabaseDto newDto = new DatabaseDto();
                 newDto.setName(name);
+                newDto.setMetadata(databaseUpdateRequestDto.getMetadata());
                 newDto.setDefinitionMetadata(databaseUpdateRequestDto.getDefinitionMetadata());
```

Comment:
A functional test is needed to check this updateDatabase() since this metadata update operation is new.

Reply:
Will do.